samples/numerics/CustomModAdd/host.py

```diff
@@ -0,0 +1,28 @@
+from Microsoft.Quantum.Numerics.Samples import CustomModAdd
+import qsharp
+
+print("Loading the numerics library...")
+# Need to load the numerics library.
+qsharp.packages.add("microsoft.quantum.numerics")
```

Comment:
Apologies for missing this in the previous round as well, but NuGet package names are generally upper-case. The current behavior of `qsharp.packages.add` is to do a case-insensitive match, but it might be good to change to upper-case in case (no pun intended) that behavior changes in the future.

```suggestion
qsharp.packages.add("Microsoft.Quantum.Numerics")
```